code/API_definitions/S2CVPN4(1).yaml

```
@@ -0,0 +1,936 @@
+openapi: "3.0.3"
+info:
+  title: Site to Cloud VPN API 
+  description: |
+    ##### Before starting to use the API, the developer needs to know about the below specified details :
+    * Site to Cloud VPN service endpoint The URL pointing to the RESTful resource of the Site to Cloud VPN API. 
+    * Authentication Security access keys such as OAuth 2.0 client credentials used by client applications to invoke the Site to Cloud VPN API.

```

Please consider the documentation within Identity and Consent Management. It is not "such as" but clearly defined to be Client Credentials.
```suggestion
    * Authentication Security access keys such as OAuth 2.0 client credentials used by client applications to invoke the Site to Cloud VPN API.
```